doc/source/user_guide.rst

```diff
@@ -0,0 +1,203 @@
+User guide
+##########
+
+Introduction
+============
+
+Python has emerged as a leading language for a diverse range of
+applications, encompassing data science, machine learning, web
+development, and workflow automation. PyAnsys is a set of Python package
+that aligns with this trend, offering a Pythonic interface to Ansys
+Products. It serves as a powerful tool for automating engineering
+workflows, conducting parametric studies, and post-processing simulation
+results, among other applications. However, configuring Python
+environments and handling dependencies can occasionally prove to be
+daunting. This is where Ansys Python Manager comes to play.
+
+What is ANSYS python manager?
+=============================
```

Comment:
```suggestion
What is the Ansys Python Manager?
=================================
```

Reply:
@RobPasMue 
vale config is not allowing it. Is there any way around for it?

Reply:
Really? Let me check that by merging